QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/BasePage.java

```diff
@@ -1,6 +1,7 @@
 package common.ui_setup.pom_setup;
 
 import common.ui_setup.SetupConstants;
+import org.openqa.selenium.By;
```

Comment:
This class should contain WebDriver as attribute and each page object class should extend it